controllers/grid/queries/form/QueryNoteForm.inc.php

```diff
@@ -141,24 +145,39 @@ public function execute(...$functionArgs)
         $query = $this->getQuery();
         if ($query->getIsClosed()) {
             $headNote = $query->getHeadNote();
-            if ($user->getId() != $headNote->getUserId()) {
+            if ($currentUser->getId() != $headNote->getUserId()) {
                 // Re-open the query.
                 $query->setIsClosed(false);
                 $queryDao = DAORegistry::getDAO('QueryDAO'); /** @var QueryDAO $queryDao */
                 $queryDao->updateObject($query);
             }
         }
 
+        $submissionId = $query->getAssocId();
+        $submission = Repo::submission()->get($submissionId);
+
+        $mailable = new MailDiscussionMessage($request->getContext(), $submission, $query);
+        $emailTemplate = Repo::emailTemplate()->getByKey($request->getContext()->getId(), $mailable::getEmailTemplateKey());
+        $context = $request->getContext();
+        $mailable->body($emailTemplate->getLocalizedData('body'))
+                ->subject('[' . $context->getLocalizedAcronym() . '] '."Re: ".$emailTemplate->getLocalizedData('subject'))
```

Comment:
We're leaving out journal acronyms from email subjects. The subject itself will be customizable using any of the email template variables, so if someone wants to include a journal prefix on all emails, they can do so by editing the subject to be `[{$acronym}] {$subject}`. But the default should just be `{$subject}`.

(Also, we can't have `Re:` in the code here. If you want to put that in, it should go into a locale key. But also it should only appear on replies, not the head note.)